OTKit/otkit-icons/README.md

```diff
@@ -0,0 +1,4 @@
+# otkit-icons
+All SVG's in this folder will be uploaded to https://cdn.otstatic.com/design-tokens/icons upon a new package publish, with each icon token set to its corresponding CDN url.
```

Comment:
interesting choice of path. I like it. As a follow up, could we move the brandon text fonts into this same root path, and then we can actually have the CDN paths available as a design token, instead of copying and pasting everywhere?

Reply:
Seems like a reasonable idea to me.  That's what I was thinking about going for /design-tokens, since I figured we may want to expand this in the future to other assets we want across consumer-web.